contrib/completion/bash/docker

```diff
@@ -407,6 +408,7 @@ __docker_complete_log_drivers() {
 
 __docker_complete_log_options() {
 	# see docs/reference/logging/index.md
+	local gcplogs_options="gcp-project labels env log-cmd"
```

Comment:
@mikedanese Two more things need to be done:

* please add `$gcplogs_options` to `$all_options` in https://github.com/mikedanese/docker/blob/gcplogs/contrib/completion/bash/docker#L420
* please add a case for `gcplogs` in https://github.com/mikedanese/docker/blob/gcplogs/contrib/completion/bash/docker#L422-L450

@thaJeztah thanks for the ping.